android-components/components/browser/engine-gecko/src/main/java/mozilla/components/browser/engine/gecko/prompt/GeckoPromptDelegate.kt

```diff
@@ -104,6 +107,38 @@ internal class GeckoPromptDelegate(private val geckoEngineSession: GeckoEngineSe
         return geckoResult
     }
 
+    override fun onSelectIdentityCredentialAccount(
+        session: GeckoSession,
+        prompt: AccountSelectorPrompt,
+    ): GeckoResult<PromptResponse>? {
+        val geckoResult = GeckoResult<PromptResponse>()
+
+        val onConfirm: (Account) -> Unit = { account ->
+            if (!prompt.isComplete) {
+                geckoResult.complete(
+                    prompt.confirm(
+                        account.id,
+                    ),
+                )
+            }
+        }
+
+        val onDismiss: () -> Unit = {
+            prompt.dismissSafely(geckoResult)
+        }
+
+        geckoEngineSession.notifyObservers {
+            onPromptRequest(
+                PromptRequest.IdentityCredential.SelectAccount(
+                    accounts = prompt.accounts.map { it.toAccount() },
+                    onConfirm = onConfirm,
+                    onDismiss = onDismiss,
+                ),
+            )
+        }
+        return geckoResult
+    }
```

Comment:
Let's add some tests to cover the behaviour :)

Reply:
Thanks for reminding me! I added the test